CRAWLING.md

```diff
@@ -1,14 +1,92 @@
 # Crawling
 
-The repository metadata shown in this portal is read from a static `repos.json` file. This project contains a [repos.json](../repos.json) file with mock data for testing and developing purposes.
+The repository metadata shown in this portal is read from a static `repos.json` file. This project contains a [repos.json](./repos.json) file with mock data, which shows the overall structure of that data.
 
-In a productive environment, consider adding an automated crawler script that fetches all InnerSource projects as outlined in the following picture.
+In a productive environment, you will likely need to run your own crawler to populate `repos.json` with the InnerSource projects that you want to show in your portal.
 
-## Step-by-step Crawling Logic
+The picture below shows the relationship between crawler and portal.
```

Comment:
Suggestion to add the hint about GitHub here as it is shown in the graphic (?): The picture below shows the relationship between crawler, portal, and repositories. We assume that the repositories are located in GitHub and use the GitHub API to fetch repository metadata. If you are using a different source code management system you can fill the structure described below with the available data.